src/javascript/_common/__tests__/tests_common.js

```diff
@@ -18,4 +18,4 @@ module.exports = {
     setURL,
     getApiToken: () => 'hhh9bfrbq0G3dRf',
     api        : new LiveApi({ websocket, appId: 1 }),
-};
+}; 
```

Comment:
The mysterious story of the gap that appeared

Reply:
ah yeah i first had it deleted and had to readd it back because it was needed lol. i'll remove the space